### PR TITLE
Change log location

### PR DESCRIPTION
**Pull request by ssoloff**
I know we said previously that it wasn't worth it to rotate the logs for the `www` subdomain because War Club was being shut down.  However, I'd like to get rid of the ~1 GiB log file (and related cruft) that's accumulated in the _/usr/share/nginx/html/tripleawarclub.org/logs/_ folder.  To that effect, this PR updates the HTTPS endpoint to use the default log location (which is already being used by the HTTP endpoint).

The last two commits are just minor non-functional cleanups.

### CODE REVIEW

File: tripleawarclub.org [hunk 1]
```diff
@@ -1,9 +1,10 @@
 server {
     listen 80;
     listen [::]:80;
-    server_name tripleawarclub.org www.tripleawarclub.org;# dice.tripleawarclub.org *.tripleawarclub.org;
+    server_name tripleawarclub.org www.tripleawarclub.org;
```
Comment (RoiEXLab) [inline tripleawarclub.org:4]: Let's encrypt now seems to support wildcard certificates.
So we could consider adding the wildcard back to this server name, if we decide to add the required DNS entries at some point.

**Review by RoiEXLab — APPROVED** [review]

File: tripleawarclub.org (hunk 1 above)
Comment (ssoloff) [reply tripleawarclub.org:4]: Note that there's already a wildcard entry on the HTTPS endpoint.  Should that be removed until we choose to support it across the board?

Comment (RoiEXLab) [reply tripleawarclub.org:4]: Probably, yeah.
I missed that one.

Comment (ssoloff) [reply tripleawarclub.org:4]: #25

**Pull request merged**